test/e2e.js

```diff
@@ -311,13 +311,22 @@ describe('plotly database connector', function Spec() {
 
     });
 
+/*
+    Loop through all dialects starting here
+*/
 
-    describe('-> normal connection UE ', () => {
+    const testedDialects = [
+        DIALECTS.MYSQL, DIALECTS.MARIADB, DIALECTS.MSSQL, DIALECTS.POSTGRES
+    ];
+
+    testedDialects.forEach(dialectUnderTest => {
+    describe(`----- ${dialectUnderTest} is being tested now -----`, () => {
+        describe('-> normal connection UE ', () => {
 
         before(openApp);
 
         describe('connecting', () => {
-            const testedDialect = DIALECTS.MYSQL;
+            const testedDialect = dialectUnderTest;
```

Comment:
can we replace all instances of `DIALECTS.MYSQL` in this test with `dialectUnderTest` now?